### ISSUE TITLE
kebab-case -> snake_case conversion not working on table names

### ISSUE THREAD

**Issue by Ramblurr** [swlkr/oksql#2]
It appears that kebab-case -> snake_case conversion is not applied to table names. 

```sql
-- resources/sql/foo_bar.clj
CREATE TABLE foo_bar (
name TEXT
);
```

```clj
; src/models/foo_bar.clj
(defn create [m]
  (oksql/insert db :foo-bar m))

(create {:name "hello"})
; => PSQLException ERROR: syntax error at or near "-"
;  Position:   org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse (QueryExecutorImpl.java:2477)
```

Loving the library by the way. So simple and elegant.

**Comment by swlkr** [comment]
Dang it! You're right! Let me fix that

**Comment by dotemacs** [comment]
Hi @swlkr,

about to give you library a spin after reading about it on Reddit.

I see that you've fixed this issue and even pushed it to Clojars, but haven't bumped up the version number in the README or closed this issue. Just a gentle ping about that... and thanks ❤️

**Comment by swlkr** [comment]
Thanks for pointing that out! Jeez, absent minded 😅. Just updated it!

**Comment by swlkr** [comment]
Closed by b7ad4c3cdca32332eac616912f93e3040d51c2ca
